x/evmutil/genesis.go

```diff
@@ -11,6 +11,8 @@ import (
 
 // InitGenesis initializes the store state from a genesis state.
 func InitGenesis(ctx sdk.Context, keeper keeper.Keeper, gs *types.GenesisState) {
+	keeper.SetParams(ctx, gs.Params)
+
```

Comment:
Any reason for setting params before genesis validation?

Reply:
yeah no reason, i will move this down